### PR TITLE
New transferred_assets() function

### PR DESCRIPTION
**Pull request by laresbernardo**
- New transferred_assets() function
- Fixed wallet_deposit_history example

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: binance
 Type: Package
 Title: Wrapper for Binance API
-Version: 0.0.1
+Version: 0.0.3
```
Comment (datawookie) [inline DESCRIPTION:4]: Please don't bump the version. I'll do that shortly.

File: R/transfer.R [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#' @inheritParams limit-5-30
+#' @param ... Additional parameters to pass they query
+#' @return A data frame.
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#' transferred_assets(type = "FUNDING_MAIN")
+#' }
+transferred_assets <- function(
```
Comment (datawookie) [inline R/transfer.R:15]: Please can you rename the function to `wallet_asset_transfer`. This is consistent with the existing naming convention which uses the section from https://binance-docs.github.io/apidocs/spot/en/ and then extracts the function name from the API endpoint.

File: R/transfer.R [hunk 3]
```diff
@@ -0,0 +1,37 @@
+#' Transferred assets among funding account and other accounts
+#'
+#' Exposes the \code{GET /sapi/v1/asset/transfer} endpoint.
+#'
+#' @inheritParams trade-parameters
+#' @inheritParams limit-5-30
+#' @param ... Additional parameters to pass they query
```
Comment (datawookie) [inline R/transfer.R:7]: Please change to "Further arguments passed to or from other methods.".

File: R/transfer.R [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    simplifyVector = FALSE,
+    security_type = "USER_DATA",
+    query = list(
+      type = type,
+      startTime = time_to_timestamp(start_time),
+      endTime = time_to_timestamp(end_time),
+      limit = limit,
+      ...
+    )
+  )[[1]]
```
Comment (datawookie) [inline R/transfer.R:36]: Is there always only a single item in the response?

File: R/wallet.R [hunk 5]
```diff
@@ -89,7 +89,7 @@ wallet_daily_snapshot <- function(
 #'
 #' @examples
 #' \dontrun{
-#' wallet_daily_snapshot("ETH")
+#' wallet_deposit_history("ETH")
```
Comment (datawookie) [inline R/wallet.R:92]: :+1: Thanks for spotting this!

**Review by datawookie — CHANGES_REQUESTED** [review]
Great. Thanks for this. Would you like to also add your name and details to the `DESCRIPTION` file?

File: DESCRIPTION (hunk 1 above)
Comment (laresbernardo) [reply DESCRIPTION:4]: Reverted on next commit

File: R/transfer.R (hunk 2 above)
Comment (laresbernardo) [reply R/transfer.R:15]: Makes sense. Changed on next commit

File: R/transfer.R (hunk 3 above)
Comment (laresbernardo) [reply R/transfer.R:7]: Changed on next commit

File: R/transfer.R (hunk 4 above)
Comment (laresbernardo) [reply R/transfer.R:36]: Haven't seen other results in this list

**Pull request merged**